src/main/scala/com/metamx/tranquility/beam/ClusteredBeam.scala

```diff
@@ -320,9 +386,80 @@ class ClusteredBeam[EventType: Timestamper, InnerBeamType <: Beam[EventType]](
     }
   }
 
+  def groupEvents(eventType: EventType, clusteredBeamMeta: ClusteredBeamMeta): DateTime = {
+    allowGranularityChange = false
+    val eventTimestamp = timestamper(eventType)
+    val eventBucket = tuning.segmentBucket(eventTimestamp)
+    // ZK does not know about any beams
```

Comment:
Is this describing what the `if` is test or is this describing the "positive" test case?  If the former, I find the wording doesn't actually portray that and I do not know what the comment is trying to say.  If the latter, move it inside the block pls.